session/session.go

```diff
@@ -1282,7 +1352,17 @@ func (s *session) ClearValue(key fmt.Stringer) {
 }
 
 // Close function does some clean work when session end.
+// Close should release the table locks which hold by the session.
 func (s *session) Close() {
+	// TODO: do clean table locks when session exited without execute Close.
+	// TODO: do clean table locks when tidb-server was `kill -9`.
+	if s.HasLockedTables() {
+		lockedTables := s.GetAllTableLocks()
+		err := domain.GetDomain(s).DDL().UnlockTables(s, lockedTables)
```

Comment:
async send ddl.

Reply:
I carefully considered, use `async clean lock` may have some problem.
In TiDB server graceful down, TiDB will close all session first, If use `async` here, TiDB server maybe exit before this `async ddl job send`.